devops/aws/deploy-playground.sh

```diff
@@ -42,5 +42,6 @@ if [ $? -eq 0 ]; then
 
   echo -e "\n\n=========== Configuring node ==========="
   ansible-playbook -i $SERVER_IP, --private-key $KEY_PATH deploy-playground-playbook.yml \
-    --extra-vars "branch_name=$BRANCH_NAME git_repo=$GIT_REPO skip_chain_setup=$SKIP_CHAIN_SETUP"
+    --extra-vars "branch_name=$BRANCH_NAME git_repo=$GIT_REPO skip_chain_setup=$SKIP_CHAIN_SETUP
+                  stack_name=$SINGLE_NODE_STACK_NAME proposal_parameters=$ALL_PROPOSALS_PARAMETERS_JSON"
```

Comment:
Do we need some quotes around "$ALL_PROPOSALS_PARAMETERS_JSON"

Reply:
Not needed as the user will have to set the env variable value with quotes

<img width="384" alt="Screenshot 2022-02-14 at 2 06 47 PM" src="https://user-images.githubusercontent.com/7795956/153828536-6fccc028-4de1-41dc-ab8e-a82eef8e1ec3.png">